lib/util/github/graphql/datasource-helper.spec.ts

```diff
@@ -382,5 +385,36 @@ describe('util/github/graphql/datasource-helper', () => {
         ]);
       });
     });
+
+    describe('Cacheable flag', () => {
+      const data = [
+        { version: v1, releaseTimestamp: t1, foo: '1' },
+        { version: v2, releaseTimestamp: t2, foo: '2' },
+        { version: v3, releaseTimestamp: t3, foo: '3' },
+      ];
+
+      test.each`
+        isPrivate | isCacheable
+        ${true}   | ${false}
+        ${false}  | ${true}
+      `(
+        'private=$isPrivate => isCacheable=$isCacheable',
+        async ({ isPrivate, isCacheable }) => {
+          httpMock
+            .scope('https://api.github.com/')
+            .post('/graphql')
+            .reply(200, resp(data, undefined, isPrivate));
+
+          const instance = new GithubGraphqlDatasourceHelper(
+            { packageName: 'foo/bar' },
+            http,
+            adapter
+          );
+          await instance.getItems();
```

Comment:
can we also expect somthing on the retun value, just in case?

Reply:
I think it's out of scope for this particular tests, but we do this in previous ones.